blog/migrations/0035_auto_20230615_2139.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 3.2.19 on 2023-06-15 21:39
+
+import common.models.helpers
+from django.db import migrations
+import wagtail.blocks
+import wagtail.fields
+import wagtail.images.blocks
+
+
+def update_lead_graphic(apps, schema_editor):
+    BlogPage = apps.get_model("blog", "BlogPage")
+
+    for page in BlogPage.objects.all():
+        stream_data = []
+
+        if page.lead_image:
+            stream_data.append(page.lead_image)
```

Comment:
This doesn't quite work and I'm having trouble figuring it out. I need to convert `CustomImage` to something that works in the streamfield and I'm not sure exactly how to do this here. @chigby do you know how to do this easily?